From 09fe4af654e7f06b45f67cc24680233c7574c6bb Mon Sep 17 00:00:00 2001
From: Davis Goodin <dagood@microsoft.com>
Date: Thu, 12 Apr 2018 21:27:19 -0500
Subject: [PATCH] Fix argsnotargets

The array argsnotargets was being treated as if it were a string, which only uses the first element.
---
 run-build.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/run-build.sh b/run-build.sh
index 61373f715..98e2c9741 100755
--- a/run-build.sh
+++ b/run-build.sh
@@ -186,7 +186,7 @@ fi
 export DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
 
 if [ $BUILD -eq 1 ]; then
-    (set -x; dotnet msbuild build.proj /bl:msbuild.generatepropsfile.binlog /p:Architecture=$ARCHITECTURE $CUSTOM_BUILD_ARGS /p:GeneratePropsFile=true /t:WriteDynamicPropsToStaticPropsFiles $argsnotargets)
+    (set -x; dotnet msbuild build.proj /bl:msbuild.generatepropsfile.binlog /p:Architecture=$ARCHITECTURE $CUSTOM_BUILD_ARGS /p:GeneratePropsFile=true /t:WriteDynamicPropsToStaticPropsFiles ${argsnotargets[@]})
     (set -x; dotnet msbuild build.proj /m /v:normal /bl:msbuild.binlog /fl /flp:v=diag /p:Architecture=$ARCHITECTURE $CUSTOM_BUILD_ARGS $args)
 else
     echo "Not building due to --nobuild"
-- 
2.16.1.windows.4

